content/docs/forms.md

```diff
@@ -182,13 +182,35 @@ class FlavorForm extends React.Component {
 
 Overall, this makes it so that `<input type="text">`, `<textarea>`, and `<select>` all work very similarly - they all accept a `value` attribute that you can use to implement a controlled component.
 
-> Note
->
-> You can pass an array into the `value` attribute, allowing you to select multiple options in a `select` tag:
->
->```js
-><select multiple={true} value={['B', 'C']}>
->```
+### Handling multiple options with `<select>`
+
+It is also possible to select multiple options within a `<select>` tag, by enabling the `multiple` attribute and passing an array into the `value` attribute. This array should contain the corresponding `value` attributes of the `<option>` tags inside the `<select>` tag.
+
+The `handleChange` method from the above example would then change to get the selected values from `event.target.selectedOptions`.
+
+```javascript{4,10-12,24}
+handleChange(event) {
```

Comment:
The indentation is inconsistent for the rest of this page, which is 2 spaces for both HTML and JS. Do fix your markup accordingly (not sure if Prettier does that for you).

Also, you can simplify `handleChange` to:

```js
const selectedFlavors = Array.from(event.target.selectedOptions).map(option => option.value);
this.setState({value: selectedFlavors});
```

Reply:
@yangshun Done. I was not sure if using arrow functions was OK for the docs or not, that is why decided to write it using `function`